### ISSUE TITLE
Rasa x does not run when domain is split in multiple files in a folder

### ISSUE THREAD

**Issue by nbeuchat** [RasaHQ/rasa#7363]
**Rasa version**: 2.1.1

**Rasa X version**: 0.33.2 (local deployment)

**Python version**: 3.7.9

**Operating system**: Ubuntu

**Issue**:
I have my domain set up in a folder as:

```
domain
|_ actions.yml
|_ slots.yml
|_ ...
```

Running `rasa x` gives me an error that the project is not a valid Rasa project and I do not have the option to specify my domain as I can do in `rasa train` for example. 

**Error (including full traceback)**:
```
This directory is not a valid Rasa project. Use 'rasa init' to create a new Rasa project or switch to a valid Rasa project directory (see https://rasa.com/docs/rasa/command-line-interface#rasa-init).
```

**Command or request that led to error**:
```
rasa x
```

**Content of domain file (domain.yml)** (if relevant):
Content is not relevant but it is split in different files in `domain` folder

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @wochinge will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by wochinge** [comment]
@degiz This will be fixed in Rasa X 0.35, right?
